core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -48,18 +50,17 @@ object Invoker {
       WhiskEntityStore.requiredProperties ++
       WhiskActivationStore.requiredProperties ++
       kafkaHost ++
+      redisHost ++
       wskApiHost ++ Map(
       dockerImageTag -> "latest",
       invokerNumCore -> "4",
       invokerCoreShare -> "2",
       invokerContainerPolicy -> "",
       invokerContainerDns -> "",
-      invokerContainerNetwork -> null)
+      invokerContainerNetwork -> null) ++
+      Map(invokerName -> null)
```

Comment:
could fold the invoker name into the previous map

```
invokerContainerNetwork -> null) ++
+      Map(invokerName -> null)
```

to 
```
invokerContainerNetwork -> null,
invokerName -> null)
```

Reply:
sure.  Do you think that is better?  The previous map seems to be all docker related things so I was keeping logical separation.  Happy to do it either way.

Reply:
Thanks for the explanation. Makes sense.